File: python-sdk/examples/smoke_test.py
from dotenv import load_dotenv

from cosevi_open_data import CoseviClient

load_dotenv()

client = CoseviClient.from_env()

print("Searching resources...")
resources = client.search_resources(query="fallecidos", limit=5)
print(resources)

print("Reading sample datastream...")
rows = client.get_datastream_data("REGIS-DE-FALLE-EN-SITIO", format="pjson", limit=5, page=1)
print(rows)

File: javascript-sdk/examples/smoke-test.ts
import "dotenv/config";
import { CoseviClient } from "../src/index.js";

const client = CoseviClient.fromEnv();

console.log("Searching resources...");
console.log(await client.searchResources({ query: "fallecidos", limit: 5 }));

console.log("Reading sample datastream...");
console.log(
  await client.getDatastreamData("REGIS-DE-FALLE-EN-SITIO", {
    format: "pjson",
    limit: 5,
    page: 1
  })
);
